### ISSUE TITLE
Still interested to maintain it?

### ISSUE THREAD

**Issue by francois2metz** [Scratch-it/analytics.js-integration-scratch-it#1]
Hi

i'm building https://bytics.io. I'm would be happy to add Scratch it / zembula to the list of available integrations.

Is your integration up to date? It yes, I can add it quickly.

**Comment by cthorner** [comment]
Yeah, sure @francois2metz . I wasn't familiar with bytics.io. Love it!
